src/watertap_contrib/reflo/costing/solar/flat_plate.py

```diff
@@ -0,0 +1,151 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pyomo.environ as pyo
+from watertap.costing.util import register_costing_parameter_block
+from watertap_contrib.reflo.costing.util import (
+    make_capital_cost_var,
+    make_fixed_operating_cost_var,
+    make_variable_operating_cost_var,
+)
+
+
+def build_flat_plate_cost_param_block(blk):
+
+    costing = blk.parent_block()
+
+    blk.cost_per_area_collector = pyo.Var(
+        initialize=600,
+        units=costing.base_currency / pyo.units.m**2,
+        bounds=(0, None),
+        doc="Cost per area for solar collector",
+    )
+
+    blk.cost_per_volume_storage = pyo.Var(
+        initialize=2000,
+        units=costing.base_currency / pyo.units.m**3,
+        bounds=(0, None),
+        doc="Cost per volume for thermal storage",
+    )
+
+    blk.contingency_frac_direct_cost = pyo.Var(
+        initialize=0,
+        units=pyo.units.dimensionless,
+        bounds=(0, 1),
+        doc="Fraction of direct costs for contingency",
+    )
+
+    blk.indirect_frac_direct_cost = pyo.Var(
+        initialize=0,
+        units=pyo.units.dimensionless,
+        bounds=(0, 1),
+        doc="Fraction of direct costs, including contingency, for indirect costs",
+    )
+
+    blk.fixed_operating_by_capacity = pyo.Var(
+        initialize=16,
+        units=costing.base_currency / (pyo.units.kW * costing.base_period),
+        bounds=(0, None),
+        doc="Fixed operating cost of flat plate plant per kW capacity",
+    )
+
+    blk.variable_operating_by_generation = pyo.Var(
+        initialize=0,
+        units=costing.base_currency / (pyo.units.MWh * costing.base_period),
+        bounds=(0, None),
+        doc="Variable operating cost of flat plate plant per MWh generated",
+    )
+
```

Comment:
I think this costing package uses the costing method from SAM, if so I guess maybe we also want to include the cost of balance of system and installation($1000 and $2500 in this case), or as a percentage of the equipment CAPEX, just to be consistent with the SAM default?

Reply:
Kurby and I had a quick chat about this. It's not in the cost function because its a very small contribution  to the capital cost compared to other components. I might be wrong but I didn't see it included in the other cost functions either. But I added land area costs similar to the PV model.